sdk/graphql-transport/src/methods.ts

```diff
@@ -625,11 +624,7 @@ export const RPC_METHODS: {
                               signAddress: 'FromAddress' in filter ? filter.FromAddress : undefined,
                               recvAddress: 'ToAddress' in filter ? filter.ToAddress : undefined,
                               kind:
-                                  'TransactionKind' in filter
-                                      ? filter.TransactionKind === 'ProgrammableTransaction'
-                                          ? TransactionBlockKindInput.ProgrammableTx
```

Comment:
Wrong type is now passed here. Noticed how before we were passing this `TransactionBlockKindInput` and now we just forward the `filter.TransactionKind`. But those types don't really match.
Since here we are building the graphQL version of the filter, `TransactionBlockKindInput` values are still what should be assigned to the `kind` field.
But now there are many more values than two, so a ternary would be too complex to use, probably we need a `switch`.